sunpy/image/transform.py

```diff
@@ -167,7 +101,176 @@ def _get_transform_method(method, use_scipy):
         try:
             import skimage  # NoQA
         except ImportError:
-            warn_user("scikit-image could not be imported. Image rotation will use scipy.")
-            method = 'scipy'
+            raise ImportError("scikit-image must be installed to be usable for rotation.")
 
     return method
+
+
+def add_rotation_function(name, *, handles_clipping=False,
+                          handles_image_nans=False, handles_nan_missing=False):
```

Comment:
How would you feel about removing the default values for these kwargs, so that every call of the decorator is explicit about what it handles. I feel like while this solution is cleaner, it's lacking some of this information when looking at the implementations as you are defaulting to this decorator doing stuff to the output?

Reply:
Good point.  If we were adding a ton of these, I think cleaner might be better than explicit, but that's not going to be the case here.  I'll make the change.

Reply:
Done